yarn.lock

```diff
@@ -2858,6 +2875,11 @@ angular-ui-bootstrap@1.3.3, angular-ui-bootstrap@^1.3.3, angular-ui-bootstrap@~1
   resolved "https://registry.yarnpkg.com/angular-ui-bootstrap/-/angular-ui-bootstrap-1.3.3.tgz#4820f895d433c17abeb4ac7387f2f122cbf8c11c"
   integrity sha1-SCD4ldQzwXq+tKxzh/LxIsv4wRw=
 
+angular-ui-bootstrap@^2.5.6:
```

Comment:
It seems that you are bringing a lot of dependencies with different version from the one used by other packages. 
This can have massive side effects, and all versions should be the same